app/globals.css

```diff
@@ -5,6 +5,7 @@
 @layer base {
   :root {
     --background: 206 28% 29%;
+    --background-hover: 206 28% 43%;
```

Comment:
Commenting on random line for thread because it's a bit unclear to me where this is coming from but I'm experiencing a possible regression here?

When I visit Arbotracker for the first time the visualizations and tables don't seem to be filled in until I select a filter. I'm guessing this PR changes the state management a bit and that's causing this?
![peek2222](https://github.com/PathoTracker/Pathotracker/assets/86806388/8d725d28-3908-4dee-a071-8047d58dd465)

Reply:
Actually maybe it's the change in `app/pathogen/arbovirus/analyze/ArboDataTable.tsx` :eyes:

The state not updating until a filter is applied and re-applied seems like maybe that ternary was keeping whatever underlying state management problem was happening here under control?